### PR TITLE
Update CapableCreateWizardletsAgent

### PR DESCRIPTION
**Pull request by KernelDeimos**

### CODE REVIEW

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js [hunk 1]
```diff
@@ -24,31 +28,74 @@ foam.CLASS({
 
   methods: [
     async function execute() {
-      var wizardlets = [];
       var capable = this.capable;
+      this.wizardlets = this.createWizardletsFromPayloads(capable.capablePayloads);
+      console.log('CAPABLE', capable);
```
Comment (tharmaman) [inline src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:33]: you intenionally leaving this logs in?

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js [hunk 2]
```diff
@@ -24,31 +28,74 @@ foam.CLASS({
...
+      console.log('WIZARDLETS', this.wizardlets);
+    },
+    function createWizardletsFromPayloads(payloads) {
+      debugger;
```
Comment (tharmaman) [inline src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:37]: left a debugger

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js [hunk 3]
```diff
@@ -24,31 +28,74 @@ foam.CLASS({
...
+      var newWizardlets = [];
+      for ( let i = 0 ; i < payloads.length ; i++ ) {
+        let capablePayload = payloads[i];
+        let wizardlet = this.createWizardletFromPayload(capablePayload);
```
Comment (tharmaman) [inline src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:41]: So this only gets fired once?

**Review by tharmaman — APPROVED** [review]

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js (hunk 1 above)
Comment (KernelDeimos) [reply src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:33]: for now yeah

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js (hunk 3 above)
Comment (KernelDeimos) [reply src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:41]: `createWizardletFromPayload` once for each Payload, but also once for each prerequisite payload

File: src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js (hunk 2 above)
Comment (KernelDeimos) [reply src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js:37]: We might need this for now

**Pull request merged**